docs/topics/settings.rst

```diff
@@ -620,6 +620,13 @@ handler (without replacement), place this in your ``settings.py``::
         'ftp': None,
     }
 
+The default HTTPS handler uses HTTP/1.1. To use HTTP/2 update
+:setting:`DOWNLOAD_HANDLERS` as follows::
+
+    DOWNLOAD_HANDLERS = {
+        'https': 'scrapy.core.downloader.handlers.http2.H2DownloadHandler',
+    }
+
```

Comment:
We should probably include a note about unencrypted HTTP/2 connections not being supported at the moment. Maybe include a reference to https://http2.github.io/faq/#does-http2-require-encryption , given that Scrapy aims to emulate browser behavior.